src/controller/controller.c

```diff
@@ -312,9 +312,10 @@ bool controller_parse_config(Controller *controller, const char *configfile) {
                 }
         }
 
-        // set logging configuration
-        bc_log_init(controller->config);
+        return true;
+}
 
+bool controller_set_config_values(Controller *controller) {
```

Comment:
Wouldn't `controller_apply_config_values` be a more descriptive name?

Reply:
Good point! Changed the function name.